### ISSUE TITLE
Google OAuth logins do not trigger account.sessions.create events correctly

### ISSUE THREAD

**Issue by dominickolbe** [appwrite/appwrite#1320]
## 🐛 Bug Report

Logging in using the Google OAuth integration creates a working session and triggers any functions with the account.sessions.create trigger, however the functions do not receive any data in the APPWRITE_FUNCTION_EVENT_DATA environment variable.

However, it is working for `account.sessions.delete`

## Have you spent some time to check if this issue has been raised before?

There is an issue and it has been solved, however it doesn't work with Google. -> https://github.com/appwrite/appwrite/issues/1176

## To Reproduce

- Enable the Google OAuth provider
- Create a Function with an account.sessions.create trigger
- Log into any account using the Google OAuth provider

## Expected behavior

The function gets the full session object in the environment variable, as it happens with normal sessions.

## Actual Behavior

```
{
  PATH: '/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin',
  HOSTNAME: '062f56145cfd',
  APPWRITE_FUNCTION_PROJECT_ID: '60c87eb670d06',
  APPWRITE_FUNCTION_TRIGGER: 'event',
  APPWRITE_FUNCTION_RUNTIME_NAME: 'Node.js',
  APPWRITE_FUNCTION_EVENT_DATA: 'null',
  APPWRITE_FUNCTION_USER_ID: 'null',
  APPWRITE_FUNCTION_JWT: 'null',
  APPWRITE_FUNCTION_DATA: 'null',
  APPWRITE_FUNCTION_ID: '60d313783cf48',
  APPWRITE_FUNCTION_NAME: 'test-function',
  APPWRITE_FUNCTION_TAG: '60d314209b4ed',
  APPWRITE_FUNCTION_RUNTIME_VERSION: '15.5',
  APPWRITE_FUNCTION_EVENT: 'account.sessions.create',
  NODE_VERSION: '15.5.1',
  YARN_VERSION: '1.22.5',
  HOME: '/root'
}
```

## Your Environment

appwrite 8.0.0 docker running on ubuntu server

**Comment by eldadfux** [comment]
Hey @dominickolbe, This issue was raised in #1176, and was fixed in #1208 - the fix is available in master and will be included in the next release.

**Comment by dominickolbe** [comment]
Awesome thanks!
